tests/unit/test_export.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+import numpy as np
+import PIL
+
+from armory.utils.export import (
+    ImageClassificationExporter,
+    ObjectDetectionExporter,
+    VideoClassificationExporter,
+    VideoTrackingExporter,
+    So2SatExporter,
+)
+
+# Mark all tests in this file as `unit`
+pytestmark = pytest.mark.unit
+
+
+def test_image_classification_export():
+    random_x = np.random.rand(32, 32, 3)
+    exporter = ImageClassificationExporter(base_output_dir="/armory/tmp/")
+    pil_img = exporter.get_sample(random_x)
+    assert isinstance(pil_img, PIL.Image.Image)
+
+
+def test_object_detection_export():
+    random_x = np.random.rand(32, 32, 3)
+    exporter = ObjectDetectionExporter(base_output_dir="/armory/tmp")
+    pil_img = exporter.get_sample(random_x)
+    assert isinstance(pil_img, PIL.Image.Image)
+
+    y = {
+        "labels": np.array([1.0]),
+        "boxes": np.array([[0.0, 0.0, 1.0, 1.0]]).astype(np.float32),
+    }
+    y_pred = {
+        "scores": np.array([1.0]),
+        "labels": np.array([1.0]),
+        "boxes": np.array([[0.0, 0.0, 1.0, 1.0]]).astype(np.float32),
+    }
+
+    pil_img_with_both_boxes = exporter.get_sample_with_boxes(random_x, y_i=y, y_i_pred=y_pred)
```

Comment:
can we change the call to something like `.get_sample(..., with_boxes=True)` so that the method call stays the same?

Reply:
working on making this change. I think the original motivation for multiple different methods was that case 1 (no boxes) only needs a single arg, whereas case 2 (with boxes) needs many more kwargs (`y`, `y_pred`, `with_boxes`, `classes_to_skip`, `score_threshold`). The `with_boxes` kwarg is almost superfluous since `y` and `y_pred` will be `None` unless you want to plot boxes. We used to (i.e. on develop we do) just effectively infer `with_boxes` from whether a `y` or `y_pred` is passed, but it feels not very transparent to the user if we don't have a kwarg explicitly stating whether boxes should be plotted

Reply:
yeah sounds good.